docs/docs/stores.md

```
@@ -29,6 +29,17 @@ These are the rules that PriceBuddy uses to extract the price, title and
 image from the product page. There is multiple ways to extract these details
 and you can mix and match different strategies.
 
+### Schema.org (JSON-LD)
+
+Probably the most robust way to extract data from a product page, this method 
+reads the JSON-LD data to extract the product detail.
+
+You can read about the spec [here](https://schema.org/Product). It will only
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)